addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -60,6 +60,7 @@ class AttachmentViewer extends Component {
          */
         this._renderedAttachment = undefined;
         this._onClickGlobal = this._onClickGlobal.bind(this);
+        this._boundOnLoadImage = this._onLoadImage.bind(this);
```

Comment:
Don't call the method "bound", it's fine to "replace" the original method, as is done with `_onClickGlobal` just above